js_modules/dagit/packages/core/src/assets/AssetEventsV2.tsx

```diff
@@ -47,7 +43,7 @@ export const AssetOverview: React.FC<Props> = ({
     loadedPartitionKeys,
     refetch,
     loading,
-  } = useRecentAssetEvents(assetKey, assetHasDefinedPartitions, params);
+  } = useRecentAssetEvents(assetKey, false, params);
```

Comment:
Un-named boolean args make me itchy -- what does `false` mean?

Reply:
Good catch going to change this now 👍